OpenChange/MAPIStoreMessage.m

```diff
@@ -178,56 +178,109 @@ - (NSDictionary *) _convertRecipient: (struct mapistore_message_recipient *) rec
                           andColumns: (struct SPropTagArray *) columns
 {
   NSMutableDictionary *recipientProperties;
+  enum MAPITAGS prop_tag;
+  char *displayName = NULL;
+  char *email = NULL;
+  uint8_t object_type = 0;
```

Comment:
Problem here is that in openchange , MAPIMAILUSER is both defined as define and enum, and I only saw the fiirst defintiion. I will change to use the enum.